Formula/docbook-xsl.rb

```diff
@@ -103,5 +104,6 @@ def post_install
     system "xmlcatalog", "#{etc}/xml/catalog", "http://docbook.sourceforge.net/release/xsl/#{version}/"
     system "xmlcatalog", "#{etc}/xml/catalog", "http://docbook.sourceforge.net/release/xsl-ns/current/"
     system "xmlcatalog", "#{etc}/xml/catalog", "http://docbook.sourceforge.net/release/xsl-ns/#{version}/"
+    system bin/"dbtoepub", "--help"
```

Comment:
Is there no better test than help

Reply:
Probably, but at the moment the challenge is to even get "--help" working. ;-)

I'll think of something.